packages/integration-react-native/package.json

```diff
@@ -22,6 +22,10 @@
   "dependencies": {
     "@loki/integration-core": "^0.25.0"
   },
+  "peerDependencies": {
```

Comment:
The reason i don't have these is because I don't want to show warnings for people using loki with regular react or with vue. You might argue that it shouldn't be in the main `loki` package, but it's there for legacy reasons. I would prefer if you removed these `peerDependency` declarations as I don't think anybody would use loki for react-native or react without also having storybook.

Reply:
In other words: your approach, is technically more correct, but I doubt it would help any users that forgot adding storybook as a dependency, but instead it would confuse a lot of users that have everything set up correctly because `loki` depends on these.

Reply:
I think a library should always declare their dependencies based on the package.json contract rather than the use cases as the latter is highly subecjtive. 

Would declare those peerDependnecies  as optional by setting `optional:true` in `peerDependenciesMeta` an option for you?

npm v6+, yarn and pnpm are all recognized this flag, https://classic.yarnpkg.com/en/docs/package-json/#toc-peerdependenciesmeta

Reply:
Ah, yes that would work!

Reply:
Fixed!

Reply:
@oblador not mean to push you for this :) just wondering if there are other concerns of this MR that I should address?

Reply:
You missed two other packages, but fixed those.